pkg/virt-launcher/monitor_test.go

```diff
@@ -45,17 +45,15 @@ var _ = Describe("VirtLauncher", func() {
 	var cmd *exec.Cmd
 	var cmdLock sync.Mutex
```

Comment:
Now, when there are no more goroutines using `cmd`, this lock actually looks redundant, dont you think?

Reply:
@vasiliy-ul  Done